packaging/debian/postinst

```diff
@@ -58,14 +57,32 @@ case "$1" in
 	    # so that we're in PATH easily
```

Comment:
I have to re-work the postinst script, I will make sure to modify this, I didn't think of that  :-)

Reply:
I mentioned to @rikysya in another PR, we can `wget -O - ${BINARIES_SERVER}/... | gunzip > /var/lib/storagenode/${binary}` since there's only one file in the zip. This way we also don't depend on the `unzip` utility.

Reply:
Will the binaries be ZIP files or gzip-compressed? `unzip` and `gunzip` don't uncompress the same types of archive.

Depending on `unzip` isn't much of a big deal in the context of a package, where it will be made available automatically if needed, but yeah, fewer dependencies is always better.

My main concern, though, is: are we sure that we will never want to put more than one file in the archive to be downloaded from the binaries server? If we make the updater depend on that quality, it could make things difficult for years to come if we ever want to change it.

Reply:
@brimstone so docker has busybox `gunzip` which doesn't seem to support any other format than `gzip`. Should `unzip/gzip` be added to dependencies as well, like with `wget`? Also, I am not sure if any other than busybox `unzip` supports reading from `stdin`?

Reply:
Adding proper gunzip (`apk -U add gzip`) makes the pipeline work, but you're right, `unzip` doesn't handle reading from stdin. Today I Learned.

Reply:
Yep, we can depend on a working gunzip! The `gzip` package is `Essential: yes` in Debian, which means you don't even need to have an explicit Depends:; it will be part of a base system.